components/merkledb/src/proof_map_index/mod.rs

```diff
@@ -202,11 +197,27 @@ impl BinaryAttribute for Option<ProofPath> {
     }
 }
 
-impl<T, K, V> ProofMapIndex<T, K, V>
+/// A Merkelized version of a map that provides proofs of existence or non-existence for the map
+/// keys.
+///
+/// `ProofMapIndex` implements a Merkle Patricia tree, storing values as leaves.
+/// `ProofMapIndex` requires that keys implement the [`BinaryKey`] trait and
+/// values implement the [`BinaryValue`] trait.
+///
+/// [`BinaryKey`]: ../trait.BinaryKey.html
+/// [`BinaryValue`]: ../trait.BinaryValue.html
+pub type ProofMapIndex<T, K, V> = ProofMapIndexBase<T, K, V, Hashed>;
+
+/// Raw variant of the `ProofMapIndex`, useful for keys that mapped directly to
+/// `ProofPath`. For example `Hash` and `PublicKey`.
+pub type RawProofMapIndex<T, K, V> = ProofMapIndexBase<T, K, V, Raw>;
```

Comment:
It may be clearest to use `Transparent` prefix instead of `Raw`.

Reply:
Not sure about this, `Raw` is much shorter.